README.md

```diff
@@ -4,79 +4,136 @@
 
 Inspired by Eric Meyers CSS Reset, Git New is a simple tool to enforce a consistent baseline Git experience for new projects.
 
-## Installation
+- [Git New](#git-new)
```

Comment:
Is the top level heading necessary?  Is it auto-generated?

Reply:
Are you referring to the ToC top level? Yes, auto-generated. It's the challenge of applying web style headings to a document style MD file. Another decision we need to make. In a document, you can have multiple heading 1's. Each one corresponding to a chapter (or similar). Mostly because a non-web document has pages, whereas a web page is, well, a page. Some of my MDs can be quite long, especially architecture, etc. I would prefer to use multiple H1s in an MD, treating it like a document that just happens to be web-renderable.

Thoughts?

Reply:
I am cool with multiple heading 1's on the page. It makes semantic sense, the same way you can have multiple H1s in HTML5 as long as they are in appropriate regions.